generated/nidcpower/session.py

```diff
@@ -1622,14 +1622,14 @@ def fetch_multiple(self, count, timeout=1.0):
 
         '''
         vi_ctype = visatype.ViSession(self._vi)  # case S110
-        channel_name_ctype = ctypes.create_string_buffer(self._repeated_capability.encode(self._encoding))  # case B520
+        channel_name_ctype = ctypes.create_string_buffer(self._repeated_capability.encode(self._encoding))  # case C010
         timeout_ctype = visatype.ViReal64(timeout)  # case S150
         count_ctype = visatype.ViInt32(count)  # case S190
         voltage_measurements_ctype = (visatype.ViReal64 * count)()  # case B600
         current_measurements_ctype = (visatype.ViReal64 * count)()  # case B600
         in_compliance_ctype = (visatype.ViBoolean * count)()  # case B600
         actual_count_ctype = visatype.ViInt32()  # case S200
-        error_code = self._library.niDCPower_FetchMultiple(vi_ctype, channel_name_ctype, timeout_ctype, count_ctype, voltage_measurements_ctype, current_measurements_ctype, in_compliance_ctype, ctypes.pointer(actual_count_ctype))
+        error_code = self._library.niDCPower_FetchMultiple(vi_ctype, channel_name_ctype, timeout_ctype, count_ctype, voltage_measurements_ctype, current_measurements_ctype, in_compliance_ctype, None if actual_count_ctype is None else (ctypes.pointer(actual_count_ctype)))
```

Comment:
Is this change intentional? It appears like it's harmless because actual_count_ctype is declared right above.

Reply:
Yes. It is needed for the `'ivi-dance'` case where we set it to None the first time.